web/src/main/java/org/openmrs/web/Listener.java

```diff
@@ -517,14 +520,16 @@ public static void loadBundledModules(ServletContext servletContext) {
 		}
 		
 		// loop over the modules and load the modules that we can
-		for (File f : folder.listFiles()) {
-			if (!f.getName().startsWith(".")) { // ignore .svn folder and the like
-				try {
-					Module mod = ModuleFactory.loadModule(f);
-					log.debug("Loaded bundled module: " + mod + " successfully");
-				}
-				catch (Exception e) {
-					log.warn("Error while trying to load bundled module " + f.getName() + "", e);
+		if (folder.listFiles() != null) {
+			for (File f : Objects.requireNonNull(folder.listFiles())) {
```

Comment:
Do you need Objects.requireNonNull when you are already sure that folder.listFiles() is not null?

Reply:
I've made two checks, because:
1) Sonar was still warning about nullable call after I made the "if" statement
2) I didn't know if this call should be guaranteed to be null (so we can throw NPE) or just ignore it by wrapping this code in (if != null) check

Reply:
Does this mean folder.listFiles() can be null on line 524?

Reply:
I think it cannot be.

Reply:
I've pushed a fix for that, leaving only the if statement